packages/ember-glimmer/lib/modifiers/action.js

```diff
@@ -194,7 +209,13 @@ export default class ActionModifierManager {
   update(modifier, element, args, dom, dynamicScope) {
     let { positional } = args;
 
-    modifier.actionName = positional.at(1).value();
+    let actionNameRef = positional.at(1);
+
+    if (actionNameRef[INVOKE]) {
+      modifier.actionName = actionNameRef;
+    } else {
+      modifier.actionName = actionNameRef.value();
+    }
```

Comment:
I don't fully understand this code yet, but the reference from `positional.at(1)` is a stable object (i.e. calling `position.at(1)` from here will give a `===` value from the same call in `install`). Like I said I need to keep reading to know what this is doing, but given the reference object itself cannot change, it's likely that you don't kneed to do any work here at all if `actionNameRef[INVOKE]` is truthy.